packages/deploy-script-support/src/coreProposalBehavior.js

```diff
@@ -75,7 +75,8 @@ export const makeCoreProposalBehavior = ({
         ? E(vatAdminSvc).getBundleIDByName(ref.bundleName)
         : ref.bundleID;
       const bundleID = await p;
-      return E(zoe).installBundleID(bundleID);
+      const label = bundleID.slice(0, 8);
+      return E(zoe).installBundleID(bundleID, label);
```

Comment:
This results in labels such as `b1-8b913`. It should arguably be the default. It can be handy in figuring out the corresponding code:

```
$ find . -name 'bundle-*' | xargs grep -l 8b913
./inter-protocol/bundles/bundle-scaledPriceAuthority.js
```

Reply:
As the default makes sense to me. Maybe different reviewers so I won't ask that it be done in this PR but that we do soon

Reply:
@FUDCo I expect it would be easy for me to change the default bundle label in general to work like this. Shall I?

Reply:
I'd also clip off the leading `b1-` before slicing, since the prefix provides almost no identification value, but other than I think this is fine idea.

Reply:
The `b1-` seems useful to say "this is the start of a bundleID; what follows is a hash". Otherwise, it's somewhat unclear that `deadbeef` is a hash while `mintHolder` is a consciously chosen name.